sherpa/models/model.py

```diff
@@ -642,22 +666,40 @@ def _set_thawed_pars_maxes(self, vals):
                 p._max = v
 
     thawedparmaxes = property(_get_thawed_par_maxes, _set_thawed_pars_maxes,
-                              doc='Access to the maximum limits for the thawed parameters')
+                              doc='The maximum limits of the thawed parameters.\n\n' +
+                              'Get or set the maximum limits of the thawed parameters\n' +
+                              'of the model as a list of numbers. If there are no\n' +
+                              'thawed parameters then [] is used. The ordering matches\n' +
+                              'that of the pars attribute.\n\n' +
+                              'See Also\n' +
+                              '--------\n' +
+                              'thawedpars, thawedarhardmaxes, thawedparmins\n')
 
     def _get_thawed_par_hardmins(self):
         return [p.hard_min for p in self.pars if not p.frozen]
 
     thawedparhardmins = property(_get_thawed_par_hardmins,
-                                 doc='The hard minimum values for the thawed parameters.')
+                                 doc='The hard minimum values for the thawed parameters.\n\n' +
+                                 'See Also\n' +
+                                 '--------\n' +
+                                 'thawedparhardmaxes, thawedparmins\n')
 
     def _get_thawed_par_hardmaxes(self):
         return [p.hard_max for p in self.pars if not p.frozen]
 
     thawedparhardmaxes = property(_get_thawed_par_hardmaxes,
-                                  doc='The hard maximum values for the thawed parameters.')
+                                  doc='The hard maximum values for the thawed parameters.\n\n' +
+                                  'See Also\n' +
+                                  '--------\n' +
+                                  'thawedparhardmins, thawedparmaxes\n')
 
     def reset(self):
-        """Reset the parameter values."""
+        """Reset the parameter values.
+
+        Restores each parameter to the last value it was set to.
```

Comment:
it was *manually* set to? It was *explicitly* set to? it was set to by calling `model.parname = x`? I know what you mean, but running fit also sets the parameters (that's why I can access them as `model.FWHM` after the fit)

Reply:
Actually, I was explicitly vague here because I don't know, and you can't easily tell from the code, the rules. I think it is all cases where you change the parameter value (so it's fits that will get reset but not explicit changes), but it relies on the reset method of the parameter which includes the mysterious `_guessed` attribute which I don't have time/energy to track down...

https://github.com/sherpa/sherpa/blob/d2668e402b4165a81ca2f49830d17a6560a26b27/sherpa/models/parameter.py#L501-L512

So I'm glad you called this out as I don't know what best to say here.

Reply:
OK, let's just leave as you have written it for now.